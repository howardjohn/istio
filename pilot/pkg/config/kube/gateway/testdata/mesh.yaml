apiVersion: networking.x-k8s.io/v1alpha1
kind: HTTPRoute
metadata:
  name: http
  namespace: default
spec:
  gateways:
    allow: FromList
    gatewayRefs:
      - name: mesh
        namespace: istio-system
  hostnames: ["first.domain.example"]
  rules:
  - matches:
    - path:
        type: Prefix
        value: /get
    forwardTo:
    - serviceName: httpbin
      port: 80
