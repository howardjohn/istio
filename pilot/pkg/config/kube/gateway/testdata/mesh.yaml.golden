apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  creationTimestamp: null
  name: http-istio-autogenerated-k8s-gateway
  namespace: default
spec:
  gateways:
  - mesh
  hosts:
  - first.domain.example
  http:
  - match:
    - uri:
        prefix: /get
    route:
    - destination:
        host: httpbin.default.svc.cluster.local
        port:
          number: 80
---
